scipy/io/wavfile.py

```diff
@@ -458,10 +458,13 @@ def _read_data_chunk(fid, format_tag, channels, bit_depth, is_big_endian,
 
         if dtype == 'V1':
             # Rearrange raw bytes into smallest compatible numpy dtype
-            dt = numpy.int32 if bytes_per_sample == 3 else numpy.int64
-            a = numpy.zeros((len(data) // bytes_per_sample, dt().itemsize),
+            dt = f'{fmt}i4' if bytes_per_sample == 3 else f'{fmt}i8'
```

Comment:
This changes `dt` from type to string, but that should be fine in the two places it's used